jetty-websocket/websocket-core/src/main/java/org/eclipse/jetty/websocket/core/internal/WebSocketChannel.java

```diff
@@ -471,6 +471,18 @@ public void demand(long n)
         connection.demand(n);
     }
 
+    @Override
+    public void suspend()
```

Comment:
See my comments on #3382, but tl;dr; suspend is a broken semantic that does not belong in core!